docs/guides/primitive-input-output.ipynb

```diff
@@ -0,0 +1,476 @@
+{
```

Comment:
<p>array of parameter bindings having shape (100, 2) -&gt; array of parameter bindings sets having shape (100,)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/documentation/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>